recipes/dolo/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "dolo" %}
+{% set version = "0.4.9.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6341cd7f45c471645f61027fc061dc9e5a113bcc7f73fb82f69c19a173dbbea1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - poetry
+  run:
+    - dolang >=0.0.8
+    - interpolation >=2.1.4
+    - ipython >=7.13.0
+    - matplotlib >=3.0
```

Depend on `-base` if you don't explicitly depend on the Qt backend of matplotlib.
```suggestion
    - matplotlib-base >=3.0
```

---

Thank you @xhochy